### PR TITLE
Add tests_to_run to github summary

### PR DESCRIPTION
**Pull request by valaparthvi**
### What does this PR do?
See what tests are run in the GH summary.
### Which issue(s) this PR fixes (optional, in fixes #<issue number>(, fixes #<issue_number>, ...) format, will close the issue(s) when PR gets merged):
Fixes #

### Checklist:
- [ ] Squashed commits into logical changes
- [ ] Documentation
- [x] GitHub Actions (if applicable) - https://github.com/rancher/hosted-providers-e2e/actions/runs/9380546741 (GKE)

### CODE REVIEW

File: .github/workflows/aks.yaml [hunk 1]
```diff
@@ -55,12 +55,7 @@ jobs:
...
       k3s_version: ${{ inputs.k3s_version || 'v1.27.9+k3s1' }}
       operator_nightly_chart: ${{ inputs.operator_nightly_chart == true || (github.event_name == 'schedule' && true) }}
-      run_p0_provisioning_tests: ${{ contains(inputs.tests_to_run, 'p0_provisioning') || (github.event_name == 'schedule' && true) }}
-      run_p0_importing_tests: ${{ contains(inputs.tests_to_run, 'p0_importing') || (github.event_name == 'schedule' && true) }}
-      run_support_matrix_provisioning_tests: ${{ contains(inputs.tests_to_run, 'support_matrix_provisioning') || (github.event_name == 'schedule' && false) }}
-      run_support_matrix_importing_tests: ${{ contains(inputs.tests_to_run, 'support_matrix_importing') || (github.event_name == 'schedule' && false) }}
-      run_k8s_chart_support_provisioning_tests: ${{ contains(inputs.tests_to_run, 'k8s_chart_support_provisioning') || (github.event_name == 'schedule' && false) }}
-      run_k8s_chart_support_importing_tests: ${{ contains(inputs.tests_to_run, 'k8s_chart_support_importing') || (github.event_name == 'schedule' && false) }}
+      tests_to_run: ${{ inputs.tests_to_run || 'p0_provisioning/p0_importing' }}
```
Comment (cpinjani) [inline .github/workflows/aks.yaml:58]: Will scheduled jobs be affected and continue to run ?

**Comment by cpinjani** [comment]
Seems job summary not added tests which ran: https://github.com/rancher/hosted-providers-e2e/actions/runs/9379101859/job/25823488088#step:27:7

**Comment by valaparthvi** [comment]
> Seems job summary not added tests which ran: https://github.com/rancher/hosted-providers-e2e/actions/runs/9379101859/job/25823488088#step:27:7

That is an old run, the issue was fixed. If you check the GKE one, the summary is seen.

File: .github/workflows/aks.yaml (hunk 1 above)
Comment (valaparthvi) [reply .github/workflows/aks.yaml:58]: They will continue to run, the default value `p0_provisioning/p0_importing` is passed, so it will be fine.

**Pull request merged**